src/lib.rs

```diff
@@ -45,7 +45,7 @@ pub enum AccountType
 }
 
 /// The current status of a price feed.
-#[derive(Copy, Clone, PartialEq, BorshSerialize, BorshDeserialize, Debug)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, BorshSerialize, BorshDeserialize)]
```

Comment:
For extra credit, you might want to consider making the derives conditional

https://stackoverflow.com/questions/42046327/conditionally-derive-based-on-feature-flag

Reply:
(The derived traits may result in more codegen occuring, but I have no evidence of real impact on build artifact size)

Reply:
I'll test it to make sure the build artifact size won't change. I don't know how smart actually rust is. I did a simple test and it didn't add anything, will test more.

Reply:
LTO is pretty good at removing these but what we could do is `#[cfg_attr(feature = "serde")]`

Reply:
I checked and binary size didn't change (in case of no usage)